hack/traffic_flow_tests.sh

```diff
@@ -27,7 +42,9 @@ temp_file=$(mktemp)
 
```

Comment:
- We will replace this file by golang when we have CDA resume working
- Can you elaborate on why we need to support setting the kubeconfig path?

Reply:
ok.